src/components/ListOwnerMessage.jsx

```diff
@@ -1,12 +1,17 @@
 import { useListOwnerDetails } from '../utils';
+import './ListOwnerMessage.css';
 
 export function ListOwnerMessage({ currentUserId, userIdFromPath, listName }) {
 	const sharedListOwner = useListOwnerDetails(userIdFromPath, listName);
 
 	if (currentUserId === userIdFromPath) {
-		return <p>You own this list.</p>;
+		return <p className="owner-message">You own this list.</p>;
 	} else if (sharedListOwner) {
-		return <p>This list belongs to {sharedListOwner.ownerName}.</p>;
+		return (
+			<p className="owner-message">
+				This list belongs to {sharedListOwner.ownerName}.
+			</p>
+		);
```

Comment:
Tiny nitpick, but I noticed if a list happens to not have an owner (`ownerName` is undefined), the message will be `This list belongs to .`

<img width="214" alt="image" src="https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/78065152/be28c034-b63d-4dfe-8694-ed20a691d032">

Reply:
Thanks Jose! Not a nitpick, a good spot! Although I know why this is happening...its because there will be lists in your account that were created before the list owner message issue was merged. That merge added a listOwner prop, and it now needs this to display the name.

We all need to clean up our FireStore before demos for this reason :-).